java/src/shared/model/player/Player.java

```diff
@@ -230,7 +237,7 @@ public void useYearOfPlenty() throws DevCardException {
      */
     @Override
     public boolean canUseRoadBuilder() {
-        return hasPlayedDevCard() ? false : developmentCardBank.canUseRoadBuilder();
+        return hasPlayedDevCard() ? false : developmentCardBank.canUseRoadBuild();
```

Comment:
A ternary operator looks a little funny here since you're using `false` directly. Probably would be better written as `return hasPlayedDevCard() && developmentCardBank.canUseRoadBuild();`